src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 4;
+    private static final int BOUND_RANDOM = 100;
+    private static final int CIRCLE_RADIUS = 10;
+    private Random randomNumber = new Random();
+    private Random randomFigure = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```
not need to create 2 objects of random